src/components/Home/Hero/HeroTitleSection.tsx

```diff
@@ -0,0 +1,55 @@
+import cn from 'classnames'
+import React from 'react'
+
+import * as styles from './styles.module.css'
+
+const HeroTitleSection = ({ className }: { className?: string }) => {
+  return (
+    <div className={className}>
+      <div
+        className={cn(
+          'text-center ',
```

Comment:
I'd like the columns to be vertically centered so that the white space in the hero feels more balanced. Something like this maybe?

```suggestion
          'text-center items-center',
```